pkg/cluster/sync.go

```diff
@@ -55,7 +54,23 @@ func (c *Cluster) Sync(newSpec *acidv1.Postgresql) error {
 	}
 
 	c.logger.Debugf("syncing volumes using %q storage resize mode", c.OpConfig.StorageResizeMode)
-	if c.OpConfig.StorageResizeMode == "pvc" {
+
+	if c.OpConfig.EnableEBSGp3Migration {
+		err = c.executeEBSMigration()
+		if nil != err {
+			return err
+		}
+	}
+
+	if c.OpConfig.StorageResizeMode == "mixed" {
```

Comment:
I fail to see the difference to `c.OpConfig.StorageResizeMode == "pvc"`. If there is another case for `StorageResizeMode`, it should be documented.

Reply:
Future work, we need "mixed" to do call to EBS for volume changes and rely on PVC to do file system extension without privileged pods.